### PR TITLE
Added default info to console help options

### LINKED ISSUE #541: Console help doesn't indicate defaults
The info displayed by nunit-console --help shows several parameters for which there are multiple options without indicating the defaults. We should tell folks!

Options affected are --process, --domain, --framework, --work.

### PR DESCRIPTION
**Pull request by rprouse**
Fixes #541 

I checked the code to double check the defaults for each of these. Please double check my defaults though :smile:

### CODE REVIEW

File: src/Common/nunit/ConsoleOptions.cs [hunk 1]
```diff
@@ -103,14 +103,14 @@ public ConsoleOptions(params string[] args)
...
+            this.Add("process=", "{PROCESS} isolation for test assemblies.\nValues: Single, Separate, Multiple. If not specified, defaults to Separate for a single assembly or Multiple for more than one.",
                 v => ProcessModel = RequiredValue(v, "--process", "Single", "Separate", "Multiple"));
 
-            this.Add("domain=", "{DOMAIN} isolation for test assemblies.\nValues: None, Single, Multiple",
+            this.Add("domain=", "{DOMAIN} isolation for test assemblies.\nValues: None, Single, Multiple. If not specified, defaults to Separate for a single assembly or Multiple for more than one.",
                 v => DomainUsage = RequiredValue(v, "--domain", "None", "Single", "Multiple"));
 
             // How to Run Tests
-            this.Add("framework=", "{FRAMEWORK} type/version to use for tests.\nExamples: mono, net-3.5, v4.0, 2.0, mono-4.0",
+            this.Add("framework=", "{FRAMEWORK} type/version to use for tests.\nExamples: mono, net-3.5, v4.0, 2.0, mono-4.0. If not specified, tests will be run under the current framework.",
```
Comment (CharliePoole) [inline src/Common/nunit/ConsoleOptions.cs]: Actually, defaults to running under the framework version for which the assembly was built. If memory serves, it now only uses the current framework if the specified or default framework is not installed.

**Comment by CharliePoole** [comment]
Pleae double-check my line comment - I wrote it out of my head.

**Comment by CharliePoole** [comment]
That is, using info from my head, not the code. I may or may not be out of my head.

File: src/Common/nunit/ConsoleOptions.cs (hunk 1 above)
Comment (rprouse) [inline src/Common/nunit/ConsoleOptions.cs]: That is what I thought too. I will find the code that I was looking at again and post a snippit here. I might have missed something.

**Comment by rprouse** [comment]
You are right about the default framework. I was looking at the code when the setting `Options.TestLoader.RuntimeSelectionEnabled` is false which uses `RuntimeFramework.CurrentFramework.ClrVersion`. I have pushed an update, so this is ready to merge if the change is good.

**Pull request merged**